android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -749,13 +749,7 @@ protected void evaluateJavascriptWithFallback(String script) {
         evaluateJavascript(script, null);
         return;
       }
-
-      try {
-        loadUrl("javascript:" + URLEncoder.encode(script, "UTF-8"));
-      } catch (UnsupportedEncodingException e) {
-        // UTF-8 should always be supported
-        throw new RuntimeException(e);
-      }
+      loadUrl("javascript:(function() {\n" + script + ";\n})();");
```

Comment:
Couldn't they simply provide this:

```
script = `; return true})() && (callSomethingElse()) && function() {`
```

... to bypass this protection?

Reply:
Yep, it's pretty trivial.